src/github.com/stellar/go-horizon/app.go

```diff
@@ -17,6 +17,8 @@ import (
 )
 
 var appContextKey = 0
+// You can override this variable using: gb build -ldflags "-X main.version aabbccdd"
+var version = "version"
```

Comment:
Seems like we should have this value default to something useful when no override occurs.  I can see two possible values:

1.  An empty string.  This would let an integrator differentiate cleanly between the "have a version" and "don't have a version" states.

2.  "unknown".  You wouldn't be able to differentiate as cleanly as 1, but the value would be acceptable for use directly in a display.